onnxscript/ir/passes/common/unused_removal_test.py

```diff
@@ -54,6 +56,59 @@ def test_remove_unused_initializers(self):
         self.assertEqual(model.graph.node[0].op_type, "Mul")
         self.assertEqual(len(model.graph.initializer), 0)
 
+    def test_remove_unused_inputs_initializers(self):
+        # remove inputs in case they are initializers
+        # if explicitly said
+        # https://github.com/microsoft/onnxscript/issues/2211
+        model = onnx.parser.parse_model(
+            """
+            <ir_version: 10, opset_import: [ "" : 17]>
+            agraph (float[N] x, float[N] two) => (float[N] z)
+            <float two = {2.0,2.0}> {
+                four = Add(two, two)
+                z = Mul(x, x)
+            }
+        """
+        )
+        model = self.remove_unused_nodes(model, True)
+        self.assertEqual(len(model.graph.node), 1)
+        self.assertEqual(model.graph.node[0].op_type, "Mul")
+        self.assertEqual(len(model.graph.inputs), 1)
```

Comment:
```suggestion
        self.assertEqual(len(model.graph.input), 1)
```
since model is a ModelProto

Reply:
Yes, done